tasks/fh-build.js

```diff
@@ -243,174 +222,162 @@ module.exports = function(grunt) {
       'get-rpm-dist-tag': {
         command: 'rpm --eval=%dist',
         options: {
-          callback: function(err, stdout, stderr, cb) {
+          callback: function (err, stdout, stderr, cb) {
             if (err || stdout[0] === '%' || stderr !== '') {
-              grunt.task.run('shell:fallback-to-non-rpm-based-distro');
-              return cb();
+              grunt.task.run('shell:fallback-to-non-rpm-based-distro')
+              return cb()
             }
-            grunt.config.set('fhos', stdout.slice(1).trim());
-            return cb();
+            grunt.config.set('fhos', stdout.slice(1).trim())
+            return cb()
           }
         }
       },
 
       'fallback-to-non-rpm-based-distro': {
         command: 'python -c "import platform; print platform.linux_distribution()"',
         options: {
-          callback: function(err, stdout, stderr, cb) {
+          callback: function (err, stdout, stderr, cb) {
             if (err) {
-              throw err;
+              throw err
             }
 
-            var dvi = stdout.toLowerCase().split("'").filter(function(v) {
-              return v !== '(' && v !== ', ' && v !== ')';
-            });
+            var dvi = stdout.toLowerCase().split("'").filter(function (v) {
+              return v !== '(' && v !== ', ' && v !== ')'
+            })
 
-            grunt.config.set('fhos', dvi[0].trim());
-            return cb();
+            grunt.config.set('fhos', dvi[0].trim())
+            return cb()
           }
         }
       },
 
       'fh-run-array': {
-        command: function(test_type) {
-          var cmd = '';
+        command: function (test_type) {
+          var cmd = ''
 
-          if (test_type) {
-            var cmdsString = grunt.template.process('<%= ' + test_type + ' %>');
-            var cmdArray = cmdsString.split(',');
-            cmd = cmdArray.join(' && ');
+          if (test_type && grunt.config.get(test_type)) {
+            var cmdsString = grunt.config.get(test_type)
+            var cmdArray = cmdsString.split(',')
+            cmd = cmdArray.join(' && ')
           } else {
-            grunt.warn("Invalid or missing parameter to " + this.nameArgs);
+            grunt.log.warn('Skipping', grunt.task.current.nameArgs,
+              '-- invalid or missing parameter')
           }
 
-          return cmd;
+          return cmd
         }
       },
 
       'fh-report-cov': {
-        command: function() {
-          var cmd = './node_modules/.bin/istanbul report';
+        command: function () {
+          var cmd = './node_modules/.bin/istanbul report'
 
-          var argsArray = Array.prototype.slice.call(arguments);
-          argsArray.forEach(function(arg) {
-            cmd += ' ' + arg;
-          });
+          var argsArray = Array.prototype.slice.call(arguments)
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) Add a semicolon at the end of this statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ASemicolon)